include/networkit/GlobalState.hpp

```diff
@@ -0,0 +1,94 @@
+#ifndef NETWORKIT_GLOBAL_STATE_HPP_
+#define NETWORKIT_GLOBAL_STATE_HPP_
+
+#ifdef NETWORKIT_STATIC_DEFINE
+#define NETWORKIT_EXPORT
+#define NETWORKIT_NO_EXPORT
+#else
+#ifndef NETWORKIT_EXPORT
+#ifdef networkit_EXPORTS
```

Comment:
Seems that we're using this definition from CMake to control the export behavior here. Is that hard coded by CMake? It does not follow our usual naming convention. I would prefer a name such as: `NETWORKIT_BUILDING_STATELIB` or something similar.